addons/pinniped/config-controller/hack/ytt/default-package-secret.yaml

```diff
@@ -0,0 +1,22 @@
+#@ load("@ytt:yaml", "yaml")
+#@ load("@ytt:data", "data")
+#@overlay/match-child-defaults missing_ok=True
+
+#@ def config():
+---
+infrastructure_provider: #@ data.values.infrastructure_provider
+identity_management_type: none
+tkg_cluster_role: workload
+#@ end
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: #@ "default-pinniped-config-" + data.values.tkr
+stringData:
+  values.yaml: #@ yaml.encode(config())
+
```

Comment:
nit: similar suggestion about this whitespace - can we stick to just one line of whitespace maximum?

Reply:
done